elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/QueryEngineTest.java

```diff
@@ -55,8 +58,20 @@ public void testFullTableLoad() {
                 .timeDimension(toProjection(playerStatsTable.getTimeDimension("recordedDate"), TimeGrain.DAY))
                 .build();
 
-        List<Object> results = StreamSupport.stream(engine.executeQuery(query).getData().spliterator(), false)
+        List<Object> results;
+        FutureTask<QueryResult> queryResult = engine.executeQuery(query);
+        queryResult.run();
```

Comment:
This code is violating DRY (do not repeat yourself).  Make a function in this class to avoid repeating the same block over and over.